### DISCUSSION TITLE
My skin's sliderend looks like sliderhead since 2024.1219.x or 2024.1224.x

### DISCUSSION THREAD

**Discussion by Glinte** [ppy/osu#31279] [Q&A] [answered]
How it looks now
![image](https://github.com/user-attachments/assets/5bbb6ab6-8777-42e6-8317-8f77af0defa9)

This is how it looked like on 2024.1208.0 (This is a screenshot from stable but I promise the sliders looks like this)
![image](https://github.com/user-attachments/assets/e0e997f2-6bfa-4ece-a95d-08ca5b8b75ca)

I have tried importing an older export of my skin into 2024.1224.1 but that doesn't work, so I believe something has changed during these two releases that causes the game to load my skin differently.

This is my skin exported from 2024.1224.1
[papeto_s skin [User] (papeto) new.zip](https://github.com/user-attachments/files/18249119/papeto_s.skin.User.papeto.new.zip)

Not sure if this would help, this is an old export of my skin, the modified date shows 2023-03-07, I have only made changes to the skin in the in-game playfield editor during 2023-2024, so the skin pngs should be identical
[papeto_s skin [User] (papeto) old.zip](https://github.com/user-attachments/files/18249170/papeto_s.skin.User.papeto.old.zip)

**Comment by peppy** [comment, marked as answer]
Your skin does not provide `hitcircle`. Please add this sprite to your skin.

**Reply by Glinte** [reply]
I guess I relied on undefined behaviour

**Reply by peppy** [reply]
Well, the skin relies on a certain fallback order which we changed to fix another bug which we changed to fix another bug. For now, we'll wait to see how often this breaks skins.

Here's the sprite you need to add:

![hitcircle@2x](https://github.com/user-attachments/assets/11e32b7e-bdd5-4e8f-a8f2-e5893a32a65f)

**Reply by Glinte** [reply]
Can confirm that adding the sprite above as `hitcircle@2x.png` in my skin fixed the problem
